README.md

```diff
@@ -121,8 +122,70 @@ service](https://console.redhat.com/openshift/create).
    ```
 
 
-1. Check all applications are synchronised
+1. Check all applications are synchronized
 
+# Factory (Edge) setup
+
+It's time to connect a "remote" factory cluster to the data center (hub).
+
+## Factory setup using the ACM UI
+
+1. Locate the kubeconfig for your factory (edge) cluster 
+
+1. Import the kubeconfig into hub (data center) cluster
+
+   Select the `Import Cluster` option beside the highlighted Create Cluster button.
+
+   On the "Import an existing cluster" page, enter the cluster name and choose Kubeconfig as the "import mode". Copy the kubeconfig details into the space provided. 
+
+   Note: After pressing `Import` you will be prompted to copy a command that can be used on the factory cluster. INGORE this message. It will go away, and is only useful if something goes wrong and you need to manually import. 
+
+## Factory setup using `cm` tool
+
+1. If you don't already have the `cm` tool then build from [here](https://github.com/open-cluster-management/cm-cli)
```

Comment:
"download from"?  I don't think we want to imply they have to build it (and a link to the repo might be read just that way)

Reply:
I'm not sure they have a download for this yet. Let me check

Reply:
Funny I was about literally going to sit down and check in with them and read this first.